org.eclipse.xtext.ui/deprecated/org/eclipse/xtext/ui/editor/copyqualifiedname/CopyQualifiedNameService.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2013, 2018 itemis AG (http://www.itemis.eu) and others.
+ * Copyright (c) 2013, 2018, 2020 itemis AG (http://www.itemis.eu) and others.
```

Comment:
Cosmetics: Usually inly two years are listed in the copyright header, e.g `2013, 2020`